core/src/main/java/io/paradoxical/cassieq/dataAccess/QueueRepositoryImpl.java

```diff
@@ -36,24 +39,34 @@ public void createQueue(final QueueDefinition definition) {
         insertQueueRecord(definition);
     }
 
-    private void insertQueueRecord(final QueueDefinition queueDefinition) {
+    @Override
+    public void setQueueStatus(@NonNull final QueueName queueName, @NonNull final QueueStatus status) {
```

Comment:
should this be a `trySetQueueStatus` because someone could set it to deleting first? if we ever have more statuses we conditional update would be nice.